code/game/gamemodes/dynamic/antag_rulesets.dm

```diff
@@ -0,0 +1,171 @@
+#define MINDSHIELDED_JOBS list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Blueshield", "Nanotrasen Representative", "Magistrate", "Internal Affairs Agent", "Nanotrasen Navy Officer", "Special Operations Officer", "Trans-Solar Federation General")
+
+/datum/ruleset
+	/// What this ruleset is called
+	var/name = "BASE RULESET"
+	/// The cost to roll this ruleset
+	var/ruleset_cost = 1
+	/// The weight to roll this ruleset
+	var/ruleset_weight = 1
+	/// The cost to roll an antagonist of this ruleset
+	var/cost = 1
+	/// The weight to roll an antagonist of this ruleset
+	var/weight = 1
```

Comment:
these are referred to as the antag cost and weight in the design doc, it may make sense to name them as such